ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priority-fee-input/index.scss

```diff
@@ -0,0 +1,4 @@
+.advanced-gas-fee-priority-fee-input {
```

Comment:
The name of the component is called PriorityFeeInput, should we use that name for consistency?